### DISCUSSION TITLE
Authentication with Access Key (Basic) at CloudFormation grain

### DISCUSSION THREAD

**Discussion by adoc-github** [QualiTorque/qualitorque.github.io#224] [Ideas]
Currently, when setting "Credentials" in the authentication section of the CloudFormation grain,
Its "Credentials" should set "Role Arn" as below.
![Credentials_setting](https://user-images.githubusercontent.com/65012009/213371710-e162826f-4b34-4398-a92b-52196984fc9b.png)

However, from the viewpoint of usability, we think that authentication by Access Key (Basic) should also be implemented.
do you like this idea?
Please consider implementing it.

**Comment by shirabendor-quali** [comment]
Hi @adoc-github ,
Thank you very much for your comment.
We will add it to our task backlog,
